source/instructors/teaching/behaviorinsights.rst

```diff
@@ -20,6 +20,8 @@ Toggle on the **Enable Behavioral Insights** setting and click **Save Changes**.
 
 Viewing Behavior Insights
 -------------------------
+.. Note:: Behavior Insights will only appear once an assignment is marked as complete.
```

Comment:
Can this be moved down a smidge to after the image and combined with the fact no indicator means lack of concern? Wording suggestion:

> Behavior Insights will only appear once an assignment is marked as complete. A lack of indicator means that there are no behavior thresholds being met - meaning the student has no indications of the specified behavior. Click on an indicator under the **Behavior** column to see the Behavior Insights Dashboard.